WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchStarterViewModelTests.swift

```diff
@@ -0,0 +1,98 @@
+
+import XCTest
+import Foundation
+import Yosemite
+import Storage
+
+@testable import WooCommerce
+
+/// Tests for `OrderSearchStarterViewModel`
+///
+final class OrderSearchStarterViewModelTests: XCTestCase {
+    private let siteID: Int64 = 1_000_000
+
+    private var storageManager: StorageManagerType!
+
+    private var storage: StorageType {
+        storageManager.viewStorage
+    }
+
+    override func setUp() {
+        storageManager = MockupStorageManager()
+        super.setUp()
```

Comment:
nit: it probably doesn't make a difference on the tests, just wondering why `super.setUp()` is called at the end whereas we usually call it first in the codebase?

Reply:
It should be the first line, yeah. Thanks for noticing that! Fixed in 8cda9201d990262bf01a460dacdd91cd9d4e1947.